skills/ios-simulator-testing/scripts/gesture.py

```
@@ -0,0 +1,334 @@
+        Returns:
+            Success status
+        """
+        # IDB doesn't have native long press, simulate with tap
+        # In real implementation, might need to use different approach
+        cmd = ["idb", "ui", "tap", str(x), str(y)]
+
+        if self.udid:
+            cmd.extend(["--udid", self.udid])
+
+        try:
+            subprocess.run(cmd, capture_output=True, check=True)
```

Ruff (0.14.1)

`subprocess` call: check for execution of untrusted input

(S603)